lib/geckoboard/field_types.rb

```diff
@@ -38,13 +52,13 @@ def to_hash
     end
   end
 
-  class MoneyField < Field
+  class MoneyField < OptionalField
     attr_reader :currency_code
 
-    def initialize(id, name: nil, currency_code: nil)
+    def initialize(id, currency_code: nil, **rest)
```

Comment:
because they're keyword arguments right?

https://flushentitypacket.github.io/ruby/2015/03/31/ruby-keyword-arguments-the-double-splat-and-starsnake.html#double-splat